packages/vscode-extension/src/debugging/RadonCDPProxyDelegate.ts

```diff
@@ -23,7 +23,9 @@ export class RadonCDPProxyDelegate implements CDPProxyDelegate {
     private sourceMapRegistry: SourceMapsRegistry,
     skipFiles: string[]
   ) {
-    this.ignoredPatterns = skipFiles.map((pattern) => new Minimatch(pattern, { flipNegate: true }));
+    this.ignoredPatterns = skipFiles.map(
+      (pattern) => new Minimatch(pattern, { flipNegate: true, dot: true })
```

Comment:
Is this change on purpose? There's no trace of it in the description

Reply:
It's necessary, because the extension directory (potentially) includes a dotted path (`.vscode` in my case), and without this option, `Minimatch` doesn't match it.